### ISSUE TITLE
Feature Request - Allow REST Post to replace feed type POST_FBA_INBOUND_CARTON_CONTENTS

### ISSUE THREAD

**Issue by kpconnell** [amzn/selling-partner-api-docs#881]
In my client's inbound shipping workflow, this step takes by far the longest and is critical path to getting labels.  The data volumes for most shipments are very small and a REST API  to be able to directly Post contents would be a huge improvement.

**Comment by johnkw** [comment]
Agreed, it does seem much slower than it should be. Also it's rather confusing and error prone. And all the XSD/XML parsing makes debugging a nightmare.

**Comment by johnkw** [comment]
Still broken. This please turn off this bot @jenilew @parvathm.

**Comment by kpconnell** [comment]
This is an important feature for high volume inbound sellers.  Using 2d barcodes and or waiting several minutes for a feed to complete for box contents does not make sense.  Box contents is not an enormous payload even for the largest supported shipments, and no other part of inbound workflow is async/feed.

**Comment by johnkw** [comment]
Turn this stupid bot off and resolve the github tickets as the actual issues are fixed. @adrian-amz @amazon-auto @chapmanjw @amvreddy-amazon @jenilew @parvathm

**Comment by Xriuk** [comment]
> and no other part of inbound workflow is async/feed.

Well, isn't there the transport estimation/confirmation waiting? So it's still async

**Comment by kpconnell** [comment]
How is this contributing to this thread?

**Comment by johnkw** [comment]
Please turn this horrible bot off and only close the github tickets as the actual issues are fixed. @adrian-amz @amazon-auto @chapmanjw @amvreddy-amazon @jenilew @parvathm

**Comment by johnkw** [comment]
Please turn this horrible bot off and only close the github tickets as the actual issues are fixed. Closing tickets that are not fixed is confusing. @adrian-amz @amazon-auto @chapmanjw @amvreddy-amazon @jenilew @parvathm
